Makefile

```diff
@@ -113,3 +113,12 @@ e2e_test: tools install
 		--offline-token=$(test_token) \
 		--openshift-version=$(openshift_version) \
 		$(NULL)
+
+.PHONY: apply_folder
+apply_folder: install
```

Comment:
Will this run the 'install' of terraform-provider?   on each apply?  it's already in image.

Reply:
Correct. 
These targets are not just for running in a container, I prefer not to assume it has been invoked before. Also, I think it's the right thing to do here, since this target is depending on a build/install. It won't hurt.

Reply:
But every apply will recompile the ocm-provider binary?  why can't u check it's there?

Reply:
Im fine with that.

I prefer a stable atomic "flow". Re-compiling is seconds.